src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -523,12 +572,27 @@ public static LocalDateTime parseLocalDateTimeForSessionsForm(String date, Strin
         }
     }
 
+    /**
+     * Convert and combine a date string, and time string of only the hour, into a LocalDateTime object.
```

Comment:
Parses a date string and a time string of only the hour into a LocalDateTime object?